packages/fast-usdc/test/pool-share-math.test.ts

```diff
@@ -72,7 +76,7 @@ test('withdrawal after deposit OK', t => {
   });
   mustMatch(proposal, shapes.withdraw);
 
-  const actual = withdrawCalc(state1, proposal);
+  const actual = withdrawCalc(state1, proposal, pDep.give);
```

Comment:
I know this already existed but I'm not sure what `pDep` means... "proposal for deposit"? It's not immediately clear what the connection is between that and the pool allocation. Could we infer it from `shareWorth` instead?

Reply:
> It's not immediately clear what the connection is between [proposal for deposit] and the pool allocation.

The connection is that `proposal.give` is what the deposit handler transfers into the pool:

https://github.com/Agoric/agoric-sdk/blob/3c74e530472f29d762b6252af5322ca8aba48a87/packages/fast-usdc/src/exos/liquidity-pool.js#L266-L269

> Could we infer it from `shareWorth` instead?

Not quite; `shareWorth.numerator` is off by 1E-6 USDC so that it never goes to zero. It's a bit of a design wart, but I haven't figured out a better way.